src/App.module.css

```diff
@@ -32,6 +61,8 @@
 }
 
 .Footer {
+  /* hide footer untill we decide what to do with it */
+  display: none;
```

Comment:
I see there is no footer in the design anymore. Need to confirm with Lirry if this is what we want across all the management screens